browser/js/dashboard/dashboard.html

```diff
@@ -1,51 +1,49 @@
 <div layout="row" class="report-snippet-wrapper" ng-cloak>
     <!-- reports / hide if im not a manager -->
-    <div class="report-container" layout="column" flex="50" ng-if="isManager">
+    <div class="report-container" layout="column" flex="50" ng-if="user.reports">
         <div class="report-title" layout="row" ng-cloak>
             Your reports
         </div>
         <md-expansion-panel-group md-component-id="report-panel" multiple>
-            <div class="parent" ng-repeat="obj in reportSnippetIds" layout="row" data-drag="true" ng-model="reportSnippetIds" data-jqyoui-options="{revert: 'invalid', helper: 'clone'}" jqyoui-draggable="{index: {{$index}}, placeholder: 'keep'}">
-                <snippet class="fromReport" id="obj.id" type="obj.type" card="false"></snippet>
+            <div ng-if="dateInRange(value)" class="parent" ng-repeat="(key, value) in user.snippets.asManager" layout="row" data-drag="true" ng-model="user.snippets.asManager" data-jqyoui-options="{revert: 'invalid', helper: 'clone'}" jqyoui-draggable="{index: {{$index}}, placeholder: 'keep'}">
+                <snippet class="fromReport" id="key" type="'report'" card="false"></snippet>
```

Comment:
Beware id inside an `ng-repeat`. Here multiple elements will end up with the same id.